backend/src/utils/error.rs

```diff
@@ -3,8 +3,22 @@ use actix_web::{error, HttpResponse, HttpResponseBuilder};
 use thiserror::Error;
 use utoipa::gen::serde_json::json;
 
+// Common error response trait
```

Comment:
No reason for comments like that, but if you really want, use `///` - doc comments